### DISCUSSION TITLE
Rule defaults while using yaml rules

### DISCUSSION THREAD

**Discussion by jobinjosem** [Azure/PSRule.Rules.Azure#1314] [Q&A] [answered]
I would like to create custom rules in YAML because I found it is easier to create and manage. At the same time, I wanted to pass parameters from ps-rule.yaml like what is used at https://azure.github.io/PSRule.Rules.Azure/setup/configuring-rules/
For example, If I wanted to check the tags on resources in Azure, but I wanted to override the values of tags that should be checked using ps-rule.yaml How it can be achieved? Any help is much appreciated.

**Comment by BernieWhite** [comment, marked as answer]
@jobinjosem Great question. Currently it's not possible to do this in YAML/ JSON-based rules. It's only possible in PowerShell-based rules.

However we'd love to add this feature. I've added it over here: microsoft/PSRule#1016

**Reply by BernieWhite** [reply]
@jobinjosem Did this answer your question? I know this is not the answer your were hoping for but please let us know. Thanks.
